packages/openapi-ts/src/types/config.d.ts

```diff
@@ -198,6 +199,17 @@ interface Input {
       include?: ReadonlyArray<string>;
     };
   };
+  fix?: {
```

Comment:
Please add a JSDoc comment about this feature so people don't have to go to docs

Reply:
I just added it. How does it look?